cartography/rules/data/rules/inactive_user_active_accounts.py

```diff
@@ -0,0 +1,47 @@
+from cartography.rules.spec.model import Fact
+from cartography.rules.spec.model import Finding
+from cartography.rules.spec.model import Maturity
+from cartography.rules.spec.model import Module
+from cartography.rules.spec.model import Rule
+
+# Facts
+_inactive_user_active_accounts_ontology = Fact(
+    id="inactive-user-active-accounts-ontology",
+    name="Active accounts linked to inactive users",
+    description="Finds user accounts that remain active despite being linked to inactive user identities.",
+    cypher_query="""
+    MATCH (u:User)-[:HAS_ACCOUNT]-(a:UserAccount)
+    WHERE COALESCE(u.inactive, False) = True
```

Comment:
This where clause makes my head spin a little bit but I get it. could be good to have a comment but it's fine